internal/portforward/service/script.go

```diff
@@ -0,0 +1,29 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func (s *Service) runPortForwardedScript(ports []uint16) (err error) {
+	// run bash script with ports as arguments
+	portStrings := make([]string, len(ports))
+	for i, port := range ports {
+		portStrings[i] = fmt.Sprint(int(port))
+	}
+	portsString := strings.Join(portStrings, " ")
+
+	scriptPath := s.settings.Scriptpath
+	s.logger.Info("running port forward script " + scriptPath)
+	cmd := exec.Command(scriptPath, portsString)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err = cmd.Run()
```

Comment:
This should be context (ctx context.Context) dependent, I think there is a exec.CommandContext for that.